dist/ExtUtils-ParseXS/lib/perlxstut.pod

```diff
@@ -606,9 +622,7 @@ with "cd"!
 	';
 	}
 
-Let's also fix the MANIFEST file so that it accurately reflects the contents
-of our extension.  The single line that says "mylib" should be replaced by
-the following three lines:
+Let's also fix the MANIFEST file by appending the following three lines:
```

Comment:
Why make this less explanatory? This is referring to the initial layout created by h2xs.

Reply:
The `mylib` is no longer detected by the initial `make manifest`. I think the first sentence should be kept, and the second sentence can be replaced by "by appending the following three lines:"

Reply:
EXAMPLE 4

`% h2xs -O -n Mytest2 Mytest2/mylib/mylib.h`

The above command generates a MANIFEST file which does not include a "mylib" line.

Reply:
```diff
  Let's also fix the MANIFEST file so that it accurately reflects the contents
- of our extension.  The single line that says "mylib" should be replaced by
+ of our extension. by appending the following three lines:
- the following three lines:
```

That looks awkward to me.

Reply:
Would this be more to your sympathies?

Let's also fix the MANIFEST file by running the following command in the top level directory.

```
% perl -MExtUtils::Manifest=mkmanifest -e 'mkmanifest()'
```

Reply:
Sorry I did not mean literally replace the second sentence. "Let's also fix the MANIFEST file so that it accurately reflects the contents of our extension by ..."

Running mkmanifest is a better suggestion than manually adding the lines, this could also be done at the end after a Makefile is generated with `make manifest`.

Reply:
No apprehension about introducing a one-liner in a .pod file otherwise free of them?

> Why make this less explanatory?

The command output would be self-explanatory. 

So you want the below change?

Let's also fix the MANIFEST file by running the following command in the top-level directory.

```
% perl -MExtUtils::Manifest=mkmanifest -e 'mkmanifest()'
```

If you want the change I'll await a response on https://github.com/Perl/perl5/pull/18240#issuecomment-719833092 before adding this change.